packages/flutter/test/material/toggle_buttons_theme_test.dart

```diff
@@ -0,0 +1,429 @@
+        .map((DiagnosticsNode node) => node.toString())
+        .toList();
+
+     expect(description, <String>[]);
+  });
+
+  testWidgets('ToggleButtonsThemeData implements debugFillProperties', (WidgetTester tester) async {
+    final DiagnosticPropertiesBuilder builder = DiagnosticPropertiesBuilder();
+    const ToggleButtonsThemeData(
+      color: Color(0xfffffff0),
+      selectedColor: Color(0xfffffff1),
+      disabledColor: Color(0xfffffff2),
+      fillColor: Color(0xfffffff3),
+      focusColor: Color(0xfffffff4),
+      highlightColor: Color(0xfffffff5),
+      hoverColor: Color(0xfffffff6),
+      splashColor: Color(0xfffffff7),
+      borderColor: Color(0xfffffff8),
+      selectedBorderColor: Color(0xfffffff9),
+      disabledBorderColor: Color(0xfffffffa),
+      borderRadius: BorderRadius.all(Radius.circular(4.0)),
+      borderWidth: 2.0,
+    ).debugFillProperties(builder);
+
+     final List<String> description = builder.properties
+        .where((DiagnosticsNode node) => !node.isFiltered(DiagnosticLevel.info))
+        .map((DiagnosticsNode node) => node.toString())
+        .toList();
+
+     expect(description, <String>[
+      'color: Color(0xfffffff0)',
+      'selectedColor: Color(0xfffffff1)',
+      'disabledColor: Color(0xfffffff2)',
+      'fillColor: Color(0xfffffff3)',
+      'focusColor: Color(0xfffffff4)',
+      'highlightColor: Color(0xfffffff5)',
+      'hoverColor: Color(0xfffffff6)',
+      'splashColor: Color(0xfffffff7)',
+      'borderColor: Color(0xfffffff8)',
+      'selectedBorderColor: Color(0xfffffff9)',
+      'disabledBorderColor: Color(0xfffffffa)',
+      'borderRadius: BorderRadius.circular(4.0)',
+      'borderWidth: 2.0',
+    ]);
+  });
+
+  testWidgets(
+    'Theme text/icon colors for enabled, selected and disabled states',
+    (WidgetTester tester) async {
+      final ThemeData theme = ThemeData();
+      const Color enabledColor = Colors.lime;
+      const Color selectedColor = Colors.green;
+      const Color disabledColor = Colors.yellow;
+
+      await tester.pumpWidget(
+        Material(
+          child: boilerplate(
+            child: ToggleButtonsTheme(
+              child: ToggleButtons(
+                color: enabledColor,
+                isSelected: const <bool>[false],
+                onPressed: (int index) {},
+                children: <Widget>[
+                  Row(children: const <Widget>[
+                    Text('First child'),
+                    Icon(Icons.check),
+                  ]),
+                ],
+              ),
+            ),
+          ),
+        ),
+      );
+
+      DefaultTextStyle textStyle;
+      IconTheme iconTheme;
+
+      // custom theme enabled color
+      expect(theme.colorScheme.onSurface, isNot(enabledColor));
+      textStyle = tester.firstWidget(
+        find.widgetWithText(DefaultTextStyle, 'First child'),
+      );
+      expect(textStyle.style.color, enabledColor);
+      iconTheme = tester.firstWidget(
```

Comment:
Turns out that there were more than one originally because `RawMaterialButton` already accounts for this case, so there were two instances of `DefaultTextStyle` and `IconTheme`. So, I simply removed the ToggleButtons' instance and updated the tests accordingly.